try {
    aadsf
} catch (error) {
    console.error(`Houve o seguinte erro :\n\n ${error}`)
}